PassengersFile.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.IO;
+
+namespace AirportTicketBooking.CSVFiles
+{
+    public static class PassengersFile
```

Comment:
the same as FlightFile and Booking File
we need to rename and separate File I/O to other class